src/test/resources/fixtures/hmh/hmh-mi/development/caliperEventGradeGraded.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id":"urn:uuid:a50ca17f-5971-47bb-8fca-4e6e6879001d",
+  "type": "GradeEvent",
+  "actor":{
+    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+    "type":"Person"
+  },
+  "action": "Graded",
+  "object": {
+    "id": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f",
+    "type": "Attempt",
+    "assignee": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+    "assignable": {
+      "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
```

Comment:
Note: I notice this field is nearly identical to object.id - with the letter "c" making it the only difference.  Just letting you know that this id should be the assessment id and should different to the object's id on line 11.
The object.id on line 11 represents the last attempt id.

See the ReadMe here https://github.com/hmhco/caliper-common-fixtures/tree/master/src/test/resources/fixtures/hmh for help commonly used Caliper fields.